### PR TITLE
TASK-57545: fix file upload using safari

### PR DESCRIPTION
**Pull request by Amine0008**
ISSUE: IOS devices use ISO-8859-1 to encode character which intoduces problems when uploading a file containing french characters (different code for non standard alphabet characters )
FIX:  used already predefined method to clean the name

### CODE REVIEW

File: apps/portlet-documents/src/main/webapp/vue-app/attachment/main.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const urls = [
   `/portal/rest/i18n/bundle/locale.portlet.attachments-${lang}.json`,
   `/portal/rest/i18n/bundle/locale.portlet.documents-${lang}.json`
 ];
-
+eXo.env.portal.utf8 = require('utf8');
```
Comment (boubaker) [inline apps/portlet-documents/src/main/webapp/vue-app/attachment/main.js]: no third party library is allowed to be imported using npm packages. You should find a solution with available APIs

File: apps/portlet-documents/package.json [hunk 2]
```diff
@@ -29,7 +29,6 @@
   "dependencies": {
     "anchorme": "^1.1.2",
     "autosize": "^4.0.2",
-    "axios": "^0.21.2",
-    "utf8": "^3.0.0"
```
Comment (boubaker) [inline apps/portlet-documents/package.json]: to revert this change please

Comment (Amine0008) [reply apps/portlet-documents/package.json]: all changes to package.json has been reverted to its  original state in the feature/maintenance branch

**Review by boubaker — APPROVED** [review]
